Palindrome/script.js

```diff
@@ -0,0 +1,20 @@
+function palindrome(str) {
+
+  let count = 0;
+  let arr = str.toLowerCase().split('').filter(char => /[a-z]/.test(char));
```

Comment:
add numbers to regexp this is not a bug. read the assignment carefully pls
--
task:
...
Note: You'll need to remove all non-alphanumeric characters (punctuation, spaces and symbols) and turn everything into the same case (lower or upper case) in order to check for palindromes.

We'll pass strings with varying formats, such as racecar, RaceCar, and race CAR among others.

We'll also pass strings with special symbols, such as 2A3*3a2, 2A3 3a2, and 2_A3*3#A2.

Reply:
Oh  Zounds, I get it now, the numbers participate in this party too. Everything will be corrected